### ISSUE TITLE
Discusson on Op: batch_normlization

### ISSUE THREAD

**Issue by seanshpark** [Samsung/ONE#1671]
@binarman wrote https://github.com/Samsung/ONE/issues/1167#issuecomment-635999572

> **batch_normlization**
  I tried to convert operator [tf.nn.batch_normalization](https://www.tensorflow.org/api_docs/python/tf/nn/batch_normalization) and toco converts it to `mul` + `add` sequence. We support both these operators. 
  **Should we mark it as already enabled?**

**Comment by seanshpark** [comment]
> Should we mark it as already enabled?

I think so :)
- lite\toco\graph_transformations\resolve_batch_normalization.cc converts BatchNorm to Mul+Add

**Comment by seanshpark** [comment]
> I think so :)

@binarman, To mark as enabled, we need to introduce a Pass that converts `BatchNorm` to `Mul` + `Add`

**Comment by seanshpark** [comment]
I think we can close this issue
